test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -1225,11 +1224,11 @@ public void PcaAnomalyTest()
 
             // REVIEW: This next test was misbehaving in a strange way that seems to have gone away
             // mysteriously (bad build?).
-            Run_TrainTest(TestLearners.PCAAnomalyDefault, TestDatasets.azureCounterUnlabeled, summary: true);
```

Comment:
>Run_TrainTest [](start = 12, length = 13)

This test doesn't test the ICanGetSummaryAsDataView or ICanGetSummaryAsIRow interface, it tests the ICanSaveSummary interface. To test the new interface implementations, please add an entry point test.

Reply:
its in a plan